.github/dependabot.yml

```diff
@@ -3,7 +3,19 @@ updates:
   # Enable version updates for Gradle
   - package-ecosystem: "gradle"
     directory: "/"
-    open-pull-requests-limit: 10
+    open-pull-requests-limit: 5
+    groups:
+      npm-major-dependencies:
+        patterns:
+          - "*"
+        update-types:
+          - "major"
```

Comment:
I would think we wouldn't want to group all major dependency updates together as those PRs would be potentially difficult/risky to merge